website/components/mdx/_getting-started-download-section.mdx

```diff
@@ -13,7 +11,7 @@ You can download the precompiled binary from [releases](https://github.com/cloud
 <Tab>
 
 ```bash
-curl -L https://versions.cloudquery.io/latest/v1/cloudquery_linux_x86_64 -o cloudquery
+curl -L https://github.com/cloudquery/cloudquery/releases/download/cli-v1.2.0-pre.1/cloudquery_Linux_x86_64 -o cloudquery
```

Comment:
We can revert these changes as we have a latest link now via Vercel redirects

Reply:
done